app/components/memorable_date_component.html.erb

```diff
@@ -1,7 +1,7 @@
-<label class="usa-label">     
+<label class="usa-label" for="doc_auth_dob">
```

Comment:
I don't see where `doc_auth_dob` is assigned? Also, because this component is meant to be reusable, I think the IDs should avoid references to specific application domains like doc auth.

Reply:
other labels on the page have for=doc_auth_[form field label]. Perhaps there's another way to accomplish this other than the way i did it explicitly here.

Reply:
Hm, those may be assigned automatically be SimpleForm? If we don't need the ID for a specific purpose, I think it'd probably be fine to leave it ou.

Reply:
ok, i removed it.